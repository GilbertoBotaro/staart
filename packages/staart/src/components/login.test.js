const React = require('react');
const {create} = require('react-test-renderer');
const Login = require('./login').default;
const {provideOothNext} = require('../../test/utils');

const LoginWithOoth = provideOothNext(Login)

describe('<Login />', () => {
    test('renders', async () => {
        const component = create(
            <LoginWithOoth/>
        )
        expect(component.toJSON()).toMatchSnapshot();
    })
});
