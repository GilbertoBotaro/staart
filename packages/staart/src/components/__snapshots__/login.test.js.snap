// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders 1`] = `
<div
  style={
    Object {
      "margin": "auto",
      "maxWidth": "300px",
    }
  }
>
  <h1>
    Log in
  </h1>
  <form
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <label
        htmlFor="username"
      >
        Username or Email
      </label>
      <input
        className="form-control"
        id="username"
        placeholder="Username"
        type="string"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        htmlFor="password"
      >
        Password
      </label>
      <input
        className="form-control"
        id="password"
        placeholder="******"
        type="password"
      />
      <a
        href="/forgot-password"
      >
        Forgot your password?
      </a>
    </div>
    <div
      className="form-group"
    >
      <button
        className="btn btn-primary btn-block"
        type="submit"
      >
        Log in
      </button>
    </div>
  </form>
  <p
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    or
  </p>
  <p>
    New user? 
    <a
      href="/register"
    >
      register
    </a>
    .
  </p>
</div>
`;
